docs/.style/homepage.sass

```diff
@@ -98,6 +103,24 @@ header
       margin: 0 1rem
       margin-bottom: 1rem
 
+.card
```

Comment:
I don't like this fix, this works by accident, if is was a `tweet` and `card` that were stacked up on mobile this might not work.

You want to change the `col-md-4` div to `col-md-4 mt-4 mt-md-0`. The second and third columns should push on mobile but not for `md` and higher.